blog/electron-24-0.md

```diff
@@ -0,0 +1,116 @@
+---
+
+Electron 24.0.0 has been released! It includes upgrades to Chromium `112`, V8 `11.2`, and Node.js `18.14.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 24.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on Twitter, or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Stack Changes
+
+- Chromium `112`
+  - [New in Chrome 112](https://developer.chrome.com/blog/new-in-chrome-112/)
+  - [New in Chrome 111](https://developer.chrome.com/blog/new-in-chrome-111/)
+  - [New in DevTools 112](https://developer.chrome.com/blog/new-in-devtools-112/)
+  - [New in DevTools 111](https://developer.chrome.com/blog/new-in-devtools-111/)
+- Node.js `18.12.1`
+  - [Node 18.12.1 blog post](https://nodejs.org/en/blog/release/v18.12.1/)
+- V8 `11.0`
+
+### Breaking Changes
+
+#### API Changed: `nativeImage.createThumbnailFromPath(path, size)`
+
+The `maxSize` parameter has been changed to `size` to reflect that the size passed in will be the size the thumbnail created. Previously, Windows would not scale the image up if it were smaller than `maxSize`, and macOS would always set the size to `maxSize`. Behavior is now the same across platforms.
+
+```js
+// a 128x128 image.
+const imagePath = path.join('path', 'to', 'capybara.png')
+
+// Scaling up a smaller image.
+const upSize = { width: 256, height: 256 }
+nativeImage.createThumbnailFromPath(imagePath, upSize).then(result => {
+  console.log(result.getSize()) // { width: 256, height: 256 }
+})
+
+// Scaling down a larger image.
+const downSize = { width: 64, height: 64 }
+nativeImage.createThumbnailFromPath(imagePath, downSize).then(result => {
+  console.log(result.getSize()) // { width: 64, height: 64 }
+})
+```
+
+#### Deprecated: `BrowserWindow.setTrafficLightPosition(position)`
+
+`BrowserWindow.setTrafficLightPosition(position)` has been deprecated, the `BrowserWindow.setWindowButtonPosition(position)` API should be used instead which accepts `null` instead of `{ x: 0, y: 0 }` to reset the position to system default.
+
+```js
+// Removed in Electron 24
+win.setTrafficLightPosition({ x: 10, y: 10 })
+win.setTrafficLightPosition({ x: 0, y: 0 })
+
+// Replace with
+win.setWindowButtonPosition({ x: 10, y: 10 })
+win.setWindowButtonPosition(null)
+```
+
+#### Deprecated: `BrowserWindow.getTrafficLightPosition()`
+
+`BrowserWindow.getTrafficLightPosition()` has been deprecated, the `BrowserWindow.getWindowButtonPosition()` API should be used instead which returns `null` instead of `{ x: 0, y: 0 }` when there is no custom position.
+
+```js
+// Removed in Electron 24
```

Comment:
```suggestion
// Deprecated in Electron 24
```

Looks like this is a verbatim copy from `breaking-changes.md`, but it should say deprecated, not removed, as it hasn't been removed yet.

Reply:
Good catch, we should update this in Breaking Changes as well